va-gov/pages/sitemap-va.xml

```diff
@@ -0,0 +1,507 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:news="http://www.google.com/schemas/sitemap-news/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml" xmlns:mobile="http://www.google.com/schemas/sitemap-mobile/1.0" xmlns:image="http://www.google.com/schemas/sitemap-image/1.1" xmlns:video="http://www.google.com/schemas/sitemap-video/1.1">
+<url> <loc>https://vabenefits.vba.va.gov/vonapp/</loc> </url>
```

Comment:
Is it odd to have subdomains in here?

Reply:
I ran the sitemap through a validator and fixed some issues with it. I have no idea about the subdomains- this was a list provided by a stakeholder.

Reply:
Yeah looks like there might be some issues with this. I'll tell @BBarone1 to ask IA / SEO to review the static sitemap we got from the VA. 
- sitemaps must use the same protocol (e.g. http or https). the sitemap we got from the va mixes these
- we can use a sitemap across hosts (e.g. www.va.gov can have a sitemap for myhealth.va.gov) if we can prove ownership to google with robots.txt- not sure exactly [what](https://www.sitemaps.org/protocol.html#sitemaps_cross_submits) this means. generally each subdomain should have its own sitemap i.e. we should remove the urls from the list that are not www.va.gov